nbproject/private/private.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unless everyone on your team uses Netbeans, you shouldn't be committing things that are IDE specific. These should go in the `.gitignore`